flask_resources/serializers/base.py

```diff
@@ -50,25 +50,24 @@ def __init__(
     ):
         """Initialize the serializer."""
         self.schema_context = schema_context
-        self.object_schema_cls = object_schema_cls
+        # TODO can be extracted to "schema_processors" but needs changes in flask_resources
```

Comment:
In order to do this, there must a be a refactor in other schemas in `flask-resources`, e.g. `BaseObjectSchema`:

https://github.com/inveniosoftware/flask-resources/blob/d849ba66b151e42e790caa57c642a0c543f6f9a3/flask_resources/parsers/schema.py#L72-L73